src/js/claims-status/utils/appeals-v2-helpers.jsx

```diff
@@ -938,13 +926,10 @@ export function getNextEvents(currentStatus, details) {
         events: [
           {
             title: `You will have your ${getHearingType(details.type)} hearing`,
+            // TODO: Should "taken" or "made" be used throughout?
             description: (
               <p>
-                Your hearing is scheduled for {formattedDate} at {details.location}. At your hearing,
-                a Veterans Law Judge will ask you questions about your appeal. A transcript of your
-                hearing will be taken and added to your appeal file. The judge will not make a
-                decision about your appeal at the hearing. Learn more about hearings, including how
-                to prepare for your hearing.
+                Your hearing is scheduled for {formattedDate} at {details.location}. At your hearing, a Veterans Law Judge will ask you questions about your appeal. A transcript of your hearing will be taken and added to your appeal file. The judge will not make a decision about your appeal at the hearing.
```

Comment:
@gnakm should taken or made be used throughout in: "A transcript of your hearing will be made/taken"? Or should both words be used in different cases?

Reply:
It should always be made.

Reply:
Good catch!

Reply:
Thanks @gnakm @cvalarida.